tests/baselines/reference/classStaticBlock8.errors.txt

```diff
@@ -0,0 +1,49 @@
+tests/cases/conformance/classes/classStaticBlock/classStaticBlock8.ts(6,21): error TS1107: Jump target cannot cross function boundary.
+tests/cases/conformance/classes/classStaticBlock/classStaticBlock8.ts(9,21): error TS1107: Jump target cannot cross function boundary.
+tests/cases/conformance/classes/classStaticBlock/classStaticBlock8.ts(12,21): error TS1107: Jump target cannot cross function boundary.
+tests/cases/conformance/classes/classStaticBlock/classStaticBlock8.ts(15,21): error TS1107: Jump target cannot cross function boundary.
+
+
+==== tests/cases/conformance/classes/classStaticBlock/classStaticBlock8.ts (4 errors) ====
+    function foo (v: number) {
+        label: while (v) {
+            class C {
+                static {
+                    if (v === 1) {

```

Comment:
Just to be sure we have adequate coverage, you should also add valid `break`/`continue` cases in a `static {}`:

```ts
class C {
  static {
    outer: break outer; // valid
    loop: while (v) {
      if (v === 1) break loop; // valid
      if (v === 2) continue loop; // valid
      if (v === 3) break; // valid
      if (v === 4) continue; // valid
    }
    switch (v) {
      default: break; // valid
    }
  }
}
```